src/ray/raylet/worker_pool.cc

```diff
@@ -921,10 +924,11 @@ std::shared_ptr<WorkerInterface> WorkerPool::PopWorker(
 
 void WorkerPool::PrestartWorkers(const TaskSpecification &task_spec,
                                  int64_t backlog_size) {
-  // Code path of task that needs a dedicated worker: an actor creation task with
-  // dynamic worker options, or any task with environment variable overrides.
+  // Code path of task that needs a dedicated worker.
```

Comment:
Hmm I think we'd eventually like to prestart workers for the same runtime env, otherwise the initial startup performance will be pretty bad. Can you just add a TODO comment here?